packages/extension-ui/src/Popup/Signing/Extrinsic.tsx

```diff
@@ -66,26 +68,38 @@ function renderMethod (data: string, { args, method }: Decoded, t: TFunction): R
     <>
       <tr>
         <td className='label'>{t<string>('method')}</td>
-        <td className='data'>
-          <details>
-            <summary>{method.section}.{method.method}{
-              method.meta
-                ? `(${method.meta.args.map(({ name }) => name).join(', ')})`
-                : ''
-            }</summary>
-            <pre>{JSON.stringify(args, null, 2)}</pre>
-          </details>
-        </td>
+        <td className='data'>{method.section}.{method.method}</td>
       </tr>
       {method.meta && (
-        <tr>
-          <td className='label'>{t<string>('info')}</td>
-          <td className='data'>
-            <details>
-              <summary>{method.meta.documentation.map((d) => d.toString().trim()).join(' ')}</summary>
-            </details>
-          </td>
-        </tr>
+        <>
+          {console.log(JSON.stringify(args, null, 2))}
+          {method.meta.args.map((arg, index) => (
+            <tr key={index}>
+              <td className='label'>{arg.name}</td>
+              <td className='data'>
+                {
+                  ADDRESS_TYPE.includes(arg.type.toHuman())
+                    ? (
+                      <Address
+                        address={args[index].Id || args[index]}
+                        isSmall={true}
+                      />
+                    )
+                    : <pre>{JSON.stringify(args[index], null, 2)}</pre>
```

Comment:
TS is also complaining about this one in a similar fashion.
> Element implicitly has an 'any' type because expression of type 'number' can't be used to index type 'string | number | true | AnyJson[] | { [index: string]: AnyJson; }'.
  No index signature with a parameter of type 'number' was found on type 'string | number | true | AnyJson[] | { [index: string]: AnyJson; }'.
  
  Wondering if I should cast it or if there's something better to do.